slatedb-bencher/benchmark-db.sh

```diff
@@ -26,7 +26,8 @@ run_bench() {
     --db-options-path $DIR/Slatedb.toml \
     --duration 60 \
     --val-len 8192 \
-    --block-cache-size 134217728 \
+    --block-cache-size 536870912 \
```

Comment:
We only run the bencher for 60s in our nightly, so the data set is fairly small. A 500 meg block cache could skew read performance quite a bit. WDYT about having a meta cache of 32mib and a block cache of 96mib? That would keep things roughly as they were, I think.